libraries/eosiolib/contracts/eosio/key_value.hpp

```diff
@@ -254,17 +254,35 @@ inline key_type make_key(double val) {
    return make_floating_key<uint64_t>(val);
 }
 
-inline key_type make_key(const char* str, size_t size, bool case_insensitive=false) {
-   size_t data_size = size + 3;
+inline key_type make_key(const char* val, size_t size, bool case_insensitive=false) {
```

Comment:
I thought we were going to kill case insensitive comparison?

Reply:
I was under the impression we were going to kill it for unicode but still support it for ASCII, but maybe that's not accurate.

Reply:
I think the final resolution was to remove it entirely.